.travis.yml

```diff
@@ -1,12 +1,17 @@
 language: node_js
 node_js:
   - 'node'
-before_script:
-  - bash <(curl -s https://raw.githubusercontent.com/wooorm/atom-travis/master/install.sh)
-  - export DISPLAY=":99"
+install:
+  - curl -s -O https://raw.githubusercontent.com/atom/ci/master/build-package.sh
+  - chmod u+x build-package.sh
+  - ./build-package.sh
```

Comment:
@wooorm FYI

Reply:
@sindresorhus Hmm? Do we need a change in atom-linter-xo?

Reply:
No, I mean, we had to stop using `wooorm/atom-travis` because it was no longer working.

Reply:
Oh sorry, I missed the context. That’s a pity! Thanks for letting me know. I’ll investigate it when I need to use it again as well 👍